_config.dev.yml

```diff
@@ -1,2 +1,2 @@
-comment_server_url: http://localhost:8000
+comment_server_url: "http://comments.savaslabs.io:51062"
```

Comment:
@kostajh Do we want to update this here? Isn't this where we can point it to our local instance of Squabble? Sorry if I'm a little behind; I haven't been following the Dockerization of things very closely.